optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/client/pages/employeeroster/EmployeeRosterPage.css

```diff
@@ -0,0 +1,493 @@
+}
+
+.viewport-frame > .viewport.horizontal > .lane > .title {
+    max-height: 22px;
+    display: inline;
+    position: absolute;
+    top: -21px;
+    padding: 0 0 0 5px;
+}
+
+.viewport-frame > .viewport.horizontal > .lane.ticks {
+    height: 30px;
+    border: none;
+    background: transparent;
+    margin: 0;
+}
+
+.viewport-frame > .viewport.horizontal > .lane.ticks > .tick {
+    transform: rotate(-60deg);
+    transform-origin: left top;
+    bottom: -10px;
+    position: absolute;
+    font-size: 0.8em;
+    opacity: 0.7;
+    z-index: 15;
+}
+
+.viewport-frame > .viewport.horizontal > .lane.ticks > .tick.harsh {
+    font-weight: bold;
+    opacity: 1;
+}
+
+.viewport-frame > .viewport.horizontal > .lane.ticks > .tick:hover {
+    color: red;
+}
+
+.viewport-frame > .viewport.horizontal > .lane.ticks > .tick:hover::before {
+    height: 1000px;
+    display: block;
+    background-color: red;
+    z-index: 100;
+    left: 16px;
+    width: 1px;
+    position: absolute;
+    opacity: 1;
+    content: "";
+    transform: rotate(60deg) translateY(30px);
+    transform-origin: inherit;
+}
+
+.viewport-frame > .viewport.horizontal > .lane > .sub-lane {
+    height: 45px;
+    border-bottom: 1px dashed #dfdfdf;
+}
+
+.viewport-frame > .viewport.horizontal > .lane > .sub-lane > .blob {
+    height: 40px;
+    border-left: 1px solid rgba(118, 133, 160, 0.1);
+    border-bottom: 1px solid rgba(118, 133, 160, 0.2);
+    border-right: 1px solid rgba(118, 133, 160, 0.1);
+    border-top: 1px solid rgba(118, 133, 160, 0.2);
+}
+
+.ui-resizable-e {
+    width: 10px;
+}
+
+.ui-resizable-s {
+    height: 10px;
+}
+
+.blob-popover {
+    position: absolute;
+    top: -20px;
+    left: -20px;
+    padding: 20px 20px 20px 20px;
+    box-sizing: content-box;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.2);
+    z-index: 20;
+}
+
+.blob-highlight-border {
+    height: 40px;
+    border-radius: 2px;
+    position: absolute;
+    background-color: rgba(255, 255, 255, 0.2);
+    border: 2px solid rgb(227, 255, 236);
+}
+
+.blob-popover-content {
+    background-color: white;
+    border: 1px solid rgba(0, 0, 0, 0.4);
+    border-radius: 4px;
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+
+.employee-roster-page {
+    position: relative;
+}
+
+.employee-roster-shift-blob {
+    font-size: 1em !important;
+    padding-top: 10px !important;
+}
+
+.last-form-group-row {
+    margin-bottom: 0;
+}
+
+.locked-for-interaction {
+    opacity: 0.7;
+    pointer-events: none;
+}
+
+/* Blobs */
+.historic {
+}
+
+.published {
+    border: 2px solid black !important;
+}
+
+.draft {
+    border: 2px dashed black !important;
+}
+
+@font-face {
+    font-family: PatternFlyIcons;
+    src: url(fonts/PatternFlyIcons-webfont.ttf);
+}
+
+.viewport-frame div {
+    box-sizing: border-box;
+}
+
+/* VIEWPORT */
+
+.viewport-frame {
+    user-select: none;
+    overflow: scroll;
+    position: relative;
+    padding: 5px;
+}
```

Comment:
This entire css file seems very long - does it duplicate anything from the spot roster css?
If so, can we pull that into a more global css file?